### PR TITLE
White label about modal. Move strings to config.

### PR DESCRIPTION
**Pull request by csduarte**
#### Summary
White label `About modal`. Move url strings to config.
These changes allow to white label the about modal based on strings on the config file.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Added or updated unit tests (required for all new features)
- [ ] All new/modified APIs include changes to [mattermost-redux](https://github.com/mattermost/mattermost-redux) (please link)
- [ ] Has UI changes
- [ ] Includes text changes and localization file updates

@jarredwitt will coordinate modifications

### CODE REVIEW

File: app/screens/about/about.js [hunk 1]
```diff
@@ -181,7 +181,7 @@ export default class About extends PureComponent {
                     <View style={style.infoContainer}>
                         <View style={style.titleContainer}>
                             <Text style={style.title}>
-                                {'Mattermost '}
+                                {`${Config.AppName} `}
```
Comment (enahum) [inline app/screens/about/about.js]: AppName is not present in the config file

File: app/screens/about/about.js [hunk 2]
```diff
@@ -207,6 +207,14 @@ export default class About extends PureComponent {
                         {licensee}
                         {learnMore}
                         <FormattedText
+                            id='mobile.about.powered_by'
```
Comment (enahum) [inline app/screens/about/about.js]: this should be shown only when the app is not built by Mattermost, and the localization string is missing in `en.json`

**Comment by jarredwitt** [comment]
@enahum changes made

File: app/screens/about/about.js [hunk 3]
```diff
@@ -206,6 +206,16 @@ export default class About extends PureComponent {
                         />
                         {licensee}
                         {learnMore}
+                        {!DeviceInfo.getBundleId().includes('mattermost') &&
```
Comment (enahum) [inline app/screens/about/about.js]: Don’t know if that’s enough cause someone can set the bundle id to something like `com.company.mattermost` and this won’t show I would be more confident if we use both of our bundle ids

**Comment by jarredwitt** [comment]
@enahum using both bundle ids now

**Pull request merged**